### PR TITLE
Use Card component for cart's sidebar

### LINKED ISSUE #1370: Consider using <Card> and <Panel> for Cart/Checkout templates
From https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1366#discussion_r357250149.

We would like to try to use `@wordpress/components` `<Card>` and maybe `<Panel>` for the Cart/Checkout blocks layout.

### LINKED ISSUE #1371: Allow importing from the last version of `@wordpress/components` regardless of what the external version is
@nerrad proposed a solution in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1366#discussion_r357300898:

> I have an idea.  I wonder if we alias `@wordpress/components` to a different package name, and use that different package name in bundles, it would allow us to import directly from the package for anything we want to use regardless of WP version (so basically what's available from the component package itself, which we can keep up to date independently)?

### PR DESCRIPTION
**Pull request by Aljullu**
This PR:
- Loads last version of `@wordpress/components` with an alias, so we can import it directly in our components (fixes #1371).
- Replaces the _Cart_ block sidebar with a `<Card>` component (fixes #1370).
- I initially thought I would need to render `<CardSeparator>` between `<RadioControl>` options, so I did a small refactor of `<RadioControl>`, then I noticed that was not required because we want borders between radio options also when not inside a `<Card>` component. :man_facepalming: Anyway, I left the refactor in this PR because I think it's useful even though it was not required.

### Screenshots

imatge

### How to test the changes in this Pull Request:

Create a post with a _Cart_ block and verify it still looks good.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,6 +107,8 @@
 		"npm": "6.11.3"
 	},
 	"dependencies": {
+		"@wordpress/components": "8.5.0",
```
Comment (Aljullu) [inline package.json]: I think we have several dependencies listed in `devDependencies` that actually belong to `dependencies`. I moved `@wordpress/components` in this PR, but if it looks good I think we could move several more.

Comment (nerrad) [reply package.json]: Yup agreed, anything that is used in production builds _directly_ (as opposed to just being included for tests) should be moved into `dependencies`.

Comment (Aljullu) [reply package.json]: Created an issue to handle that: #1424.

File: assets/js/base/components/radio-control/index.js [hunk 2]
```diff
@@ -16,95 +16,20 @@ const RadioControl = ( {
...
-									{ secondaryDescription }
-								</Label>
-							) }
-						</label>
-					)
-				) }
+				{ options.map( ( option ) => (
+					<RadioControlOption
+						key={ `${ id }-${ option.value }` }
+						name={ id }
```
Comment (nerrad) [inline assets/js/base/components/radio-control/index.js]: Are we sure this `name` will be unique? Should it have an instance-id added to it?

File: assets/js/base/components/radio-control/option.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+		description,
+		secondaryLabel,
+		secondaryDescription,
+	} = option;
+	const onChangeValue = ( event ) => onChange( event.target.value );
+
+	return (
+		<label
+			key={ `${ name }-${ value }` }
+			className="wc-blocks-radio-control__option"
```
Comment (nerrad) [inline assets/js/base/components/radio-control/option.js]: should the class name be `wc-block` as per changes you've made elsewhere?

**Comment by Aljullu** [comment]
> would you be able to do a size comparison before and after on the cart files as well as the various `vendor` files?  Before committing to this, it'd be good to see exactly what the impact is on size for the record.

Sure! This table compares `master` with this branch, so it includes the changes from #1411:

| File | `master` size | This branch size
| --- | ---: | ---:
| `style.css` | 75,71KB | 76,11KB
| `vendors-style.css` | - | 3,75KB

All other CSS files have the same size.

**Pull request merged**